oam-dev/kubevela
# Feat: support addon definition in cue format

### Description of your changes

Now our definition in addons are all YAML, This PR support definition's CUE format.

This makes it possible to develop addon definition with `vela def`



Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [ ] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 

### How has this code been tested

1. Add one test.: render an addon with a cue format definition.
2. test by hand
  - test cue definition
  ![image](https://user-images.githubusercontent.com/47812250/145148919-be2ec768-fc77-4240-8f8c-c98e04fcf1bc.png)
  - enable addon    
    ![image](https://user-images.githubusercontent.com/47812250/145149390-8ced8163-9a3b-4a52-945e-46bb1983e9ff.png)
  - check addon
    ![image](https://user-images.githubusercontent.com/47812250/145149464-4b4faed1-7a88-4e8f-8551-7b04bd58956a.png)
 




### Special notes for your reviewer
